sentinel-core/src/main/java/com/alibaba/csp/sentinel/annotation/SentinelResource.java

```diff
@@ -77,6 +79,16 @@
      */
     String defaultFallback() default "";
 
+
+    /**
+     * The {@code globalFallback} is used as the global fallback handler class.
+     * The returned class should implement the {@link IGlobalFallback} interface.
+     * If no specific global fallback handler is provided, the default {@link DefaultGlobalFallback} class will be used.
+     * @return The class that serves as the global fallback handler.
+     */
+    Class<? extends IGlobalFallback> globalFallback() default DefaultGlobalFallback.class;
```

Comment:
IMO, the scenario the related issue(#3110) aims to solve is to have a global fallback even when neither fallback nor defaultFallback is configured in the annotations. So, the global fallback should be configured in the construction method of SentinelResourceAspect, rather than through annotations (like what defaultFallback did).